transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -514,4 +529,62 @@ void updateHelpText() {
         }
         helpText.getParent().layout();
     }
+
+    void updateManifestPackageImports(IProject project,
+                                      File pomFile) throws Exception {
+        IPath pomPath = project.getRawLocation() != null
+                        ? project.getRawLocation()
+                        : ResourcesPlugin.getWorkspace().getRoot().getLocation().append(project.getFullPath());
+        Model pomModel = MavenPlugin.getMaven().readModel(new File(pomPath.append("pom.xml").toOSString()));  //$NON-NLS-1$
+        Build build = pomModel.getBuild();
+        Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+        Plugin plugin = pluginsByName.get("org.apache.felix:maven-bundle-plugin"); //$NON-NLS-1$
+        if (plugin == null) {
+            plugin = new Plugin();
+            plugin.setGroupId("org.apache.felix"); //$NON-NLS-1$
+            plugin.setArtifactId("maven-bundle-plugin"); //$NON-NLS-1$
+            plugin.setVersion("3.2.0"); //$NON-NLS-1$
+            plugin.setExtensions(true);
+            build.addPlugin(plugin);
+        }
+        if (!(plugin.getConfiguration() instanceof Xpp3Dom)) throw new Exception(Messages.TransformationEditor_invalidPomConfiguration);
```

Comment:
So, I originally added this to address @apupier earlier comment about an unchecked type cast, but after researching this more, it appears that the only possible instance of the configuration will be a Xpp3Dom, so the original type cast should have been okay.  The suggestion you provided in the fork, @lhein, allows for an "additional" configuration if the instance isn't what's expected, and since that's not possible, I'm going to revert back to the blind cast.  For what it's worth, all examples of working with the configuration that I can find online do the same thing.